packages/lit/src/components/wizard-token-field/index.ts

```diff
@@ -0,0 +1,74 @@
+import type { ColorToken, DimensionToken, FontFamilyToken } from "@nl-design-system-community/design-tokens-schema";
+import { html, LitElement, nothing } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { DesignToken } from "style-dictionary/types";
+
+const tag = 'wizard-token-field';
+
+// Declare the custom element for TypeScript
+declare global {
+  interface HTMLElementTagNameMap {
+    [tag]: WizardTokenField;
+  }
+}
+
+@customElement(tag)
+export class WizardTokenField extends LitElement {
+  @property() token: ColorToken | DimensionToken | FontFamilyToken | DesignToken = {};
+  @property() path: string = '';
+  @property() options = [];
+  @property({ type: Number }) depth = 0;
+
+  static maxDepth = 3;
+
+  get entries() {
+    const entries = Object.entries(this.token).filter(([key]) => key[0] !== '$');
+    return entries;
+  }
+
+  get inputType() {
+    switch(this?.token?.$type) {
+      case 'color': return 'color';
+      case 'dimension': return 'number';
+      case 'fontfamily': return 'string';
+      case 'fontfamilies': return 'string';
```

Comment:
```suggestion
      case 'fontfamily':
      case 'fontfamilies': return 'text';
```

Zal waarschijnlijk ook goed gaan omdat `input[type=string]` degrade naar `input[type=text]`, maar toch.
Is dit evt in een `type` te vatten?